client/src/pages/admin/dashboard.tsx

```diff
@@ -1,15 +1,134 @@
+                  <Listbox.Button className="relative w-full cursor-default rounded-lg bg-white py-1 pl-3 pr-10 text-left shadow-md focus:outline-none focus-visible:border-indigo-500 focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-orange-300 sm:text-sm">
+                    <span className="block truncate cursor-pointer">{selected.batch}</span>
+                    <span className="pointer-events-none absolute inset-y-0 right-0 flex items-center pr-2">
+                      <ChevronUpDownIcon
+                        className="h-5 w-5 text-gray-400"
+                        aria-hidden="true"
+                      />
+                    </span>
+                  </Listbox.Button>
+                  <Transition
+                    as={Fragment}
+                    leave="transition ease-in duration-100"
+                    leaveFrom="opacity-100"
+                    leaveTo="opacity-0"
+                  >
+                    <Listbox.Options className="z-50 absolute mt-1 max-h-60 w-full overflow-auto rounded-md bg-white py-1 text-base shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none sm:text-sm">
+                      {years.map((year, yearIdx) => (
+                        <Listbox.Option
+                          key={yearIdx}
+                          className={({ active }) =>
+                            `relative select-none py-2 pl-10 pr-4 ${active ? 'bg-sats-10 text-slate-50' : 'text-gray-900'
+                            }`
+                          }
+                          value={year}
+                        >
+                          {({ selected }) => (
+                            <>
+                              <span
+                                className={`block text-left truncate ${selected ? 'font-medium' : 'font-normal'
+                                  }`}
+                              >
+                                {year.batch}
+                              </span>
+                              {selected ? (
+                                <span className="absolute inset-y-0 left-0 flex items-center pl-3 slate-10">
+                                  <CheckIcon className="h-5 w-5" aria-hidden="true" />
+                                </span>
+                              ) : null}
+                            </>
+                          )}
+                        </Listbox.Option>
+                      ))}
+                    </Listbox.Options>
+                  </Transition>
+                </div>
+              </Listbox>
+            </div>
+          </div>
+          <Chart
+            chartType="BarChart"
+            width="100%"
+            height="100%"
+            data={data}
+            options={options}
+          />
+        </Card>
+      </div>
+    </AdminLayout >
   )
 }
 
-export default Dashboard
+export default Dashboard
```

Comment:
Add New Line

Reply:
I added the line in my other PRs, sorry I just noticed the comment. 
Just to avoid conflict when rebasing, I added it in task [SATS-304]